packages/core/src/Layout/LayoutManager.ts

```diff
@@ -69,18 +73,14 @@ class LayoutManager {
       return pageContent;
     }
 
-    return this.layouts[name].insertPage(pageContent, pageNav, pageIncludedFiles);
+    return this.layouts[name].insertPage(pageContent, pageNav, new Set<string>(pageIncludedFiles));
```

Comment:
Why do we create a new instance of `pageIncludedFiles` here? Is it to avoid passing by reference?